code/modules/clothing/head/armor.dm

```diff
@@ -80,7 +80,7 @@
 
 /obj/item/clothing/head/armor/helmet/technomancer
 	name = "insulated technomancer helmet"
-	desc = "A piece of armor used in hostile work conditions to protect the head. Comes with a built-in flashlight."
+	desc = "A piece of armor used in hostile work conditions to protect the head. Comes with a built-in flashlight. The appearance of the visor can be changed with a wrench."
```

Comment:
```suggestion
	desc = "A piece of armor used in hostile work conditions to protect the head. Comes with a built-in flashlight."
	description_info = "Appearance of the visor can be changed by wrenching."
```
Out-of-character gameplay advices should be kept separate from in-character item info.
Variables such as `description_info` and `description_antag` exist for this purpose.